src/ensembl/stories/shared-components/chevron/Chevron.stories.tsx

```diff
@@ -0,0 +1,140 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useState } from 'react';
+
+import Chevron, {
+  Direction as ChevronDirection
+} from 'src/shared/components/chevron/Chevron';
+import RadioGroup from 'src/shared/components/radio-group/RadioGroup';
+
+import styles from './Chevron.stories.scss';
+
+export const ChevronStory = () => {
+  const [direction, setDirection] = useState('down');
```

Comment:
How about making `direction` to be of type `ChevronDirection`?

Reply:
You mean `useState<ChevronDirection>`? Done. Still need to narrow down the type in the onChange function for the RadioGroup though:

![image](https://user-images.githubusercontent.com/6834224/124840711-803b1e80-df83-11eb-9c6b-b8b07d250be8.png)